src/BatchClient.php

```diff
@@ -15,7 +15,7 @@
  *
  * @author Joel Wurtz <jwurtz@jolicode.com>
  */
-class BatchClient implements HttpClient
+final class BatchClient implements HttpClient
```

Comment:
Batch client does not have an abstraction layer for all of it's methods.

Reply:
:+1: maybe we can add final with annotation like Symfony does

Reply:
added an interface